DigitalLearningSolutions.Web/ViewModels/Common/SearchablePage/BasePaginatedViewModel.cs

```diff
@@ -37,6 +38,10 @@ protected BasePaginatedViewModel(PaginationResult<T> paginationResult)
 
         public int ItemsPerPage { get; set; }
 
+        public int TotalItems { get; set; }

```

Comment:
`TotalItems` is only used to set `JavascriptSearchSortFilterPaginateEnabled`. You could make it a property and assign it directly in the constructor.